src/main/java/frc/robot/Constants.java

```diff
@@ -68,7 +68,7 @@ public static final class FrontLeft { // Module 2
 
         public static final double STEER_OFFSET =
             IS_COMP_BOT
-                ? -Math.toRadians(30.498) // comp bot offset
+                ? -Math.toRadians(331.41632080078125) // comp bot offset
```

Comment:
Why are you setting offsets in a feature? This is fine, but it's better as a fix branch -- it can be merged independently of your feature, so it should be independent.